src/shared/helpers/getActualDataList.test.ts

```diff
@@ -0,0 +1,108 @@
+import { describe, expect, it } from 'vitest';
+import { getActualDataList } from './getActualDataList';
+import { dayJS } from '@/app/services/dayjs';
+import { Course } from '@/app/types';
+import { isCourse } from '@/entities/courses/helpers/is-course';
+import { EventCardProps } from '@/entities/events';
+
+const staleAfterDays = 14;
```

Comment:
We already have such a constant 
![image](https://github.com/rolling-scopes/site/assets/114234698/e26c4ad8-579a-4236-bf61-75f4afccb1a3)

Reply:
If I'll use a global cons here and later someone will desided to change it,  it can break test logic